### PR TITLE
Update to environment.yml (conda-forge support)

### PR DESCRIPTION
**Pull request by dafrose**
opened PR as suggested in issue #4853 
- removed pyqt version requirement (as mayavi and pysurfer require pyqt 4 and not 5)
- moved everything possible to conda instead of pip
- removed clinicalgraphics channel and added conda-forge channel instead

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -26,19 +26,19 @@ dependencies:
 - numpydoc
 - flake8
 - spyder
+- nitime
+- nibabel
+- nilearn
```
Comment (agramfort) [inline environment.yml]: I would not put nitime, nibabel, pysurfer, etc. as conda packages. They are up to date on pypi but not necessarily on conda AFAIK. Please only move mayavi related stuff (traits*, pyface, etc.)

**Comment by dafrose** [comment]
is there a reason to require pyqt 5 from within mne?

**Comment by agramfort** [comment]
pyqt5 is the current recent backend on matplotlib and should work with
latest mayavi (I think). If we can we should use it.

**Comment by dafrose** [comment]
I see. Makes sense. I haven't checked the dependencies of other packages, but `mayavi` requires `pyqt4` on conda. [The docs](http://docs.enthought.com/mayavi/mayavi/installation.html#requirements-for-manual-installs), however, mention it as an "optional requirement", whatever that means.

**Comment by mmagnuski** [comment]
The mayavi from the official conda channel may indeed not work that well with pyqt5, I was using mayavi from other channels most of the time (current `environment.yml` uses mayavi from github via pip) - but this is worth checking.

**Comment by larsoner** [comment]
Mayavi can also use wx but we should use the qt5 one

**Comment by dafrose** [comment]
I am still quite new to continuous integration and all this... but how can codecov by my (rather minor) changes drop so low?

**Comment by agramfort** [comment]
don't worry too much on codecov but focus on travis

**Comment by larsoner** [comment]
@dafrose feel free to look at #5012 for changes that (presumably) supersede these

**Pull request closed without merging**